### PR TITLE
get inner exception message while sending github http request

### PR DESCRIPTION
**Pull request by OsmondJiang**
image

### CODE REVIEW

**Comment by OsmondJiang** [comment]
image

File: src/docfx/lib/github/GitHubAccessor.cs [hunk 1]
```diff
@@ -117,7 +117,7 @@ public async Task<(Error, GitHubUser)> GetUserByLogin(string login)
             }
             catch (Exception ex)
             {
-                return (Errors.GitHubApiFailed(_url, ex.Message), null);
+                return (Errors.GitHubApiFailed(_url, ex.InnerException?.Message ?? ex.Message), null);
```
Comment (yufeih) [inline src/docfx/lib/github/GitHubAccessor.cs]: can you write these detailed diagnostic info using Log.Write?, the Error message is mainly for writers, not for troubleshooting

Comment (OsmondJiang) [reply src/docfx/lib/github/GitHubAccessor.cs]: so what the message do you want to return to the end user?

Comment (yufeih) [reply src/docfx/lib/github/GitHubAccessor.cs]:
```suggestion
                Log.Write(ex);
                return (Errors.GitHubApiFailed(_url, ex.Message), null);
```

Comment (OsmondJiang) [reply src/docfx/lib/github/GitHubAccessor.cs]: if we already have the really message, why not directly return to end user? let's say we are using by some developers, they can find more info from here. And yes, we can also write to disgnostic.

Comment (yufeih) [reply src/docfx/lib/github/GitHubAccessor.cs]: sure

**Pull request merged**